doc/debugging.rst

```diff
@@ -14,33 +14,91 @@ This allows breakpoints and the execution flow to be viewed and controlled right
 Debugging with ipdb
 ===================
 
-Spyder offers the following debugging features integrated into the native GUI:
+You can fully control debugger execution from the :guilabel:`Debug` menu, :guilabel:`Debug toolbar` and via configurable keyboard shortcuts, along with the standard ``ipdb`` `console commands`_.
+Additionally, the  :doc:`editor` shows the line of code the debugger is currently stopped on with an arrow.
 
-* Multiple means of setting and clearing normal and conditional breakpoints for any line in a file opened in the :doc:`editor`.
+.. _console commands: https://pythonconquerstheuniverse.wordpress.com/2009/09/10/debugging-in-python/
 
-  * By selecting the respective option from the Debug menu.
-  * Through pressing a configurable keyboard shortcut (:kbd:`F12` for normal, or :kbd:`Shift-F12` for conditional breakpoints by default).
-  * By double-clicking to the left of the line number in an open file.
-  * With an ``ipdb.set_trace()`` statement in your code (after ``import`` ing ``pdb``).
-  * Interactively, using the ``b`` command in an ``ipdb`` session.
+GIF: Inset of Spyder's Editor, with a breakpoint set and the condition dialog open (commands from the toolbar)
```

Comment:
I think we agreed that at this point we are going to show a gif with

* A breakpoint set in the editor
* Start the debugger from the toolbar
* Press the `next line` button once to show the blue arrow on the next line after the breakpoint.

Reply:
I don't recall the specifics on this, so I'm not sure...